week1/react-exercises/src/3-counter/Counter.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+
+const Counter = () => {
+  const [count, setCount] = useState(0);
+
+  const increaseCount = () => setCount(count + 1);
+
+  const feedback = count > 10 ? "It`s higher than 10!" : "Keep counting...";
+
+  return (
+    <div className="counter">
+      <h1>Counter:</h1>
+      <Count currCount={count} />
+      <Button addOne={increaseCount} />
+      <h3>{feedback}</h3>
+    </div>
+  );
+};
+
+const Count = props => <h2>{props.currCount}</h2>;
+
+const Button = props => <button onClick={props.addOne}>Add 1!</button>;
```

Comment:
- Use de-structuring for your props
- Place this component in a separate file

```suggestion
const Button = ({ addOne }) => <button onClick={addOne}>Add 1!</button>;
```